coderd/rbac/regosql/configs.go

```diff
@@ -36,6 +36,21 @@ func TemplateConverter() *sqltypes.VariableConverter {
 	return matcher
 }
 
+func AuditLogConverter() *sqltypes.VariableConverter {
+	matcher := sqltypes.NewVariableConverter().RegisterMatcher(
+		resourceIDMatcher(),
+		sqltypes.StringVarMatcher("COALESCE(audit_logs.organization_id :: text, '')", []string{"input", "object", "org_owner"}),
+		// Templates have no user owner, only owner by an organization.
```

Comment:
Copy-pasta? This is talking about audit logs, not templates.

Reply:
This should be audit logs :+1: